index.js

```diff
@@ -0,0 +1,93 @@
+const hamburger = document.querySelector('.menu-icon');
+const menu = document.querySelector('.mobile-menu');
+const closeIcon = document.querySelector('.fa-close');
+const moreBtn = document.querySelector('.more-btn');
+const featureBox = document.querySelector('.featured-container');
+
+hamburger.addEventListener('click', () => {
+  menu.style.display = 'block';
+});
+
+closeIcon.addEventListener('click', () => {
+  menu.style.display = 'none';
+});
+
+const featuredSpeakers = [
+  {
+    name: 'Cynthia Bunmi',
+    img: './assets/images/cynthia.jpg',
+    alt: 'Photo of Cynthia Bunmi',
+    position: 'Manager at Premium sites Nig. Ltd',
+    details: `Cynthia is an expert in Gender and Development and is well-known
+    for her book "Education isn't given but acquired"`,
+  },
+  {
+    name: 'Alexa Thorne',
+    img: './assets/images/alexa.jpg',
+    alt: 'Photo of Alexa Thorne',
+    position: 'CEO at Bones Teller',
+    details: `Alexa is an expert in Gender and Development and is well-known
+    for her book "Education isn't given but acquired"`,
+  },
+  {
+    name: 'George Maxin',
+    img: './assets/images/georgee.jpg',
+    alt: 'Photo of George Maxin',
+    position: 'Professor in Business Management',
+    details: `George is an expert in Gender and Development and is well-known
+    for her book "Education isn't given but acquired"`,
+  },
+  {
+    name: 'Olayinka Adeoye',
+    img: './assets/images/adeoyee.jpg',
+    alt: 'Photo of Olayinka Adeoye',
+    position: 'Manager at Premium sites Nig. Ltd',
+    details: `Olayinka is an expert in Gender and Development and is well-known
+    for her book "Education isn't given but acquired"`,
+  },
+  {
+    name: 'Nwachi Sarah',
+    img: './assets/images/nwachi.jpg',
+    alt: 'Photo of Nwachi Sarah',
+    position: 'Manager at Premium sites Nig. Ltd',
+    details: `Sarah is an expert in Gender and Development and is well-known
+    for her book "Education isn't given but acquired"`,
+  },
+  {
+    name: 'Tommy Agnes',
+    img: './assets/images/agnes.jpg',
+    alt: 'Photo of Tommy Agnes',
+    position: 'Manager at Premium sites Nig. Ltd',
+    details: `Agnes is an expert in Gender and Development and is well-known
+    for her book "Education isn't given but acquired"`,
+  },
+];
+
+const allCodes = featuredSpeakers.map((item) => (
+  `
+  <section>
+    <div class="featured-img">
+      <img src="${item.img}" alt="${item.alt}">
+    </div>
+    <article class="featured-text">
+      <h3>${item.name}</h3>
+      <p>${item.position}</p>
+      <hr>
+      <p>${item.details}</p>
+    </article>
+  </section>`
```

Comment:
- [ ] To make a great match with the original template for a better touch and feel of your page, you may consider adding the colored background image to your speakers as is the case in the original template 
### Original design :point_down: 
![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/e6d98762-6231-47b9-9bf6-ff867642d2d6)
### Your design :point_down: 
![image](https://github.com/Wineshuga/EmpowerHER-Foundation/assets/97100091/2012b829-f6bf-4faf-82e9-6e1421dd6176)